hybrid/per_func_comp_main.c

```diff
@@ -0,0 +1,257 @@
+/***
+ * This example implements a simple cheri type allocator to show
+ * how cheri otype values are determined.
+ * Note: cheriintrin.h is not used in this example.
+ ***/
+
+#include "../include/print.h"
+#include "include/utils.h"
+#include <sys/sysctl.h>
+#include <sys/types.h>
+
+#define SANDBOX_CODE_SZ 1000 // approx. later modify it.
```

Comment:
Is this still to be modified, or is set? If the former, can we add a `TODO` to make it clear?

Reply:
1. 
I think there's a bit of cleanup left, and, as Laurie mentioned, I believe this is not yet executed by buildbot. To do that, you'll need to add the extra test in our test script [1]. I assume you want a run OK on morello-hybrid? If so, throw it in the set of tests from line 84 onwards.
---> it passed builbot for morello-hybrid. will check 

2) Otherwise, I think I understand the intuition of this example, but missing some background. As far as I'm aware, otypes are used primarily for sealing. If that's the case, I wonder if we're breaking some assembly-level assumptions with this approach.
--> Specify mechanism how it would break assembly-level assumptions with this approach. 

3) I'm also missing where (if anywhere) we're doing the transition / invocation / validation.
---> As the comment in a head describes, and I said at the meeting three times, this is to show a generated value of the otype.

Reply:
Can add "TODO"